Demos/communication/include/SignalHandler.hpp

```diff
@@ -255,3 +255,7 @@ void RegisterSignalHandler(SignalHandler handler)
 {
     gSignalMonitor.reset(new SignalMonitor(std::move(handler)));
 }
+void ShutdownSignalHandler()
```

Comment:
Nur noch eine Leerzeile hier 👾 

```suggestion

void ShutdownSignalHandler()
```